pkg/k8s/node.go

```diff
@@ -65,6 +66,12 @@ func ParseNode(k8sNode *slim_corev1.Node, source source.Source) *nodeTypes.Node
 			continue
 		}
 
+		if _, typeFound := addressTypes[addr.Type]; typeFound {
+			scopedLog.Errorf("detected multiple IPs of the same type %s for node %s and Cilium is going to only consider the first one", addr.Type, k8sNode.Name)
```

Comment:
Could you make this log msg like this?

```suggestion
			scopedLog.WithLogFields(logrus.Fields{
			  logfields.Node: k8sNode.Name,
			  logfields.Type: addr.Type
			}).Warn("Detected multiple IPs of the same address type, Cilium will only consider the first IP in the Node resource")
```